### PR TITLE
fix: [CDS-58615]: Input set Ref in Trigger: UI breaking when editing …

### PR DESCRIPTION
**Pull request by pankajsingh-harness**
…Trigger yaml

### Summary
This PR has the fix for the Empty Screen in the Pipeline Input Tab in trigger creation when inputSet is removed from the YAML and user moves to the Visual View.

This PR also has fix for [https://harness.atlassian.net/browse/CDS-58663](https://harness.atlassian.net/browse/CDS-58663)

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/72-triggers/components/Triggers/ScheduledTrigger/ScheduledTriggerWizard.tsx [hunk 1]
```diff
@@ -776,6 +776,8 @@ export default function ScheduledTriggerWizard(
     }
 
     const runPipelineFormErrors = isNewGitSyncRemotePipeline
+      ? null
+      : formikProps.values.inputSetRefs?.length
```
Comment (PratyushGar) [inline src/modules/72-triggers/components/Triggers/ScheduledTrigger/ScheduledTriggerWizard.tsx:780]:
```suggestion
     const runPipelineFormErrors = isNewGitSyncRemotePipeline || formikProps.values.inputSetRefs?.length
  ? null
  : await getFormErrors({
      latestPipeline: latestPipelineFromYamlView || latestPipeline,
      latestYamlTemplate: yamlTemplate,
      orgPipeline: values.pipeline,
      setSubmitting
    });
```

Comment (pankajsingh-harness) [reply src/modules/72-triggers/components/Triggers/ScheduledTrigger/ScheduledTriggerWizard.tsx:780]: Done

**Review by PratyushGar — APPROVED** [review]

**Comment by pankajsingh-harness** [comment]
retrigger cypress-cd

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Comment by pankajsingh-harness** [comment]
retrigger cypress-cv

**Comment by pankajsingh-harness** [comment]
retrigger cypress-cv

**Review by PratyushGar — APPROVED** [review]

**Comment by pankajsingh-harness** [comment]
`Prometheus.spec.ts` from `SorryCypressCV-Cypress_CV ` and coverage is failing.

**Pull request merged**